homework8/task5.js

```diff
@@ -0,0 +1,56 @@
+const options = {
+  method: "GET",
+  url: "https://matchilling-chuck-norris-jokes-v1.p.rapidapi.com/jokes/random",
+  headers: {
+    accept: "application/json",
+    "X-RapidAPI-Key": "330ade6295msha04dc868820df5cp1d10f2jsn4dcbddab3bda",
+    "X-RapidAPI-Host": "matchilling-chuck-norris-jokes-v1.p.rapidapi.com",
+  },
+};
+
+function getRandom() {
+  return fetch(
+    "https://matchilling-chuck-norris-jokes-v1.p.rapidapi.com/jokes/random",
```

Comment:
also you can create separate value for URL "https://matchilling-chuck-norris-jokes-v1.p.rapidapi.com/jokes, because u have 3 places where u re-use it

Reply:
fixed